addons/im_livechat/static/tests/call.test.js

```diff
@@ -0,0 +1,40 @@
+import {
+    click,
+    contains,
+    mockGetMedia,
+    setupChatHub,
+    start,
+    startServer,
+} from "@mail/../tests/mail_test_helpers";
+
+import { defineLivechatModels } from "@im_livechat/../tests/livechat_test_helpers";
+
+import { test } from "@odoo/hoot";
+import { mockDate } from "@odoo/hoot-mock";
+
+import { Command, serverState } from "@web/../tests/web_test_helpers";
+
+defineLivechatModels();
+
+test("should display started a call message", async () => {
+    mockDate("2025-01-01 12:00:00", +1);
+    mockGetMedia();
+    const pyEnv = await startServer();
+    const guestId = pyEnv["mail.guest"].create({ name: "Visitor #" });
+    const channelId = pyEnv["discuss.channel"].create({
+        channel_member_ids: [
+            Command.create({ partner_id: serverState.partnerId }),
+            Command.create({ guest_id: guestId }),
+        ],
+        channel_type: "livechat",
+        livechat_active: true,
+        livechat_operator_id: serverState.partnerId,
+    });
+    setupChatHub({ opened: [channelId] });
+    await start();
+    await contains(".o-mail-ChatWindow", { text: "Visitor #" });
+    await click("[title='Start a Call']");
+    await contains(".o-mail-NotificationMessage", {
+        text: `${serverState.partnerName} started a call.1:00 PM`,
+    });
```

Comment:
This test passes even without the fix. It just needs the missing code in mock server which is not really the fix itself.

If you make test with `livechat_username` and check it uses this `livechat_username` then the test will fail before fix and pass after fix.